CompilerForCAP/gap/InferDataTypes.gi

```diff
@@ -897,7 +897,7 @@ CapJitAddTypeSignature( "QUO_INT", [ IsInt, IsInt ], IsInt );
 CapJitAddTypeSignature( "IsZero", [ IsInt ], IsBool );
 CapJitAddTypeSignature( "OneImmutable", [ IsIntegers ], IsInt );
 CapJitAddTypeSignature( "^", [ IsPerm, IsInt ], IsPerm );
-CapJitAddTypeSignature( "PermList", [ IsList ], IsPerm );
+#CapJitAddTypeSignature( "PermList", [ IsList ], IsPerm );
```

Comment:
I suggest we keep this: We know that it works and `PermList` will not change anyway.

Reply:
Done.